src/main/webapp/js/results/results.js

```diff
@@ -0,0 +1,101 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/* ==========================================================================
+   RE-ROLL
+   ========================================================================== */
+function roll() {
+    const pickEl = document.getElementById("pick");
+    const infoEl = document.getElementById("info");
+    fetch(`/query`, { method: "GET" })
```

Comment:
see comment above about unifying this same call

Reply:
roll() and query() are basically the same, and they appear as new code. Let's fix the duplication.